cache-config/testing/ort-tests/traffic_ops_ort_test.go

```diff
@@ -75,16 +86,8 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
-	log.Infof(`Using Config values:
-			   TO Config File:       %s
-			   TO Fixtures:          %s
-			   TO URL:               %s
-			   TO Session Timeout In Secs:  %d
-			   DB Server:            %s
-			   DB User:              %s
-			   DB Name:              %s
-			   DB Ssl:               %t
-               UseIMS:               %v`, *configFileName, *tcFixturesFileName, tcd.Config.TrafficOps.URL, tcd.Config.Default.Session.TimeoutInSecs, tcd.Config.TrafficOpsDB.Hostname, tcd.Config.TrafficOpsDB.User, tcd.Config.TrafficOpsDB.Name, tcd.Config.TrafficOpsDB.SSL, tcd.Config.UseIMS)
+	// TODO: does this test suite need logging?
```

Comment:
For unit tests, I might say no; but for the Integration Test Framework, which involves a large, long, slow Docker setup, IMO it's useful for debugging to have it.

Reply:
If you say so. It's only being used in 9 places (8 of which are executed a maximum of once and the remaining one is spammy and should possibly be removed IMO), so I just wasn't sure the timestamps and output redirection that the library offers are providing much benefit over `fmt.Fprintf(os.Stderr, ...)`, but it's also not hurting anything, so if literally anyone finds it worthwhile then it's not worth reworking. I'll remove the TODO.